### PR TITLE
480 update thrust force mapping documentation and testing

### PR DESCRIPTION
**Pull request by JulianHammerl**
* **Tickets addressed:** MAXGNC-480
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

A change was made to the thrForceMapping module to allow 3-axis attitude control with DV thrusters (previously it was assumed that this is not possible and only 2 axes could be controlled). This change was done without updating the documentation. This ticket updates the documentation and unit test.

## Verification

The unit test passes.

## Documentation

Documentation was updated.

## Future work

N/A

### CODE REVIEW

File: src/fswAlgorithms/effectorInterfaces/thrForceMapping/thrForceMapping.rst
Comment (patkenneally) [inline src/fswAlgorithms/effectorInterfaces/thrForceMapping/thrForceMapping.rst]: :large_blue_circle: Did you mean "nominal thruster state on" rather than "in"?

**Review by patkenneally — APPROVED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

File: src/fswAlgorithms/effectorInterfaces/thrForceMapping/thrForceMapping.rst
Comment (JulianHammerl) [reply src/fswAlgorithms/effectorInterfaces/thrForceMapping/thrForceMapping.rst]: Yes. Fixed.

**Pull request merged**
